From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 15 Jul 2020 12:05:12 +0200
Subject: ANDROID: GKI: build in more Bluetooth drivers

Multiple platforms need these, so build them in so we don't need to
support an api we don't really need to.

Bug: 157965270
Bug: 162909109
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: Icc99311bd349e226cf61deba98414d5e48750c54
Signed-off-by: Connor O'Brien <connoro@google.com>
[joneslee: Removed defconfig elements]
Signed-off-by: Lee Jones <lee.jones@linaro.org>
---
 arch/arm64/configs/db845c_gki.fragment | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -19,8 +19,6 @@ CONFIG_MFD_SPMI_PMIC=m
 CONFIG_SPMI_MSM_PMIC_ARB=m
 CONFIG_REGULATOR_QCOM_RPMH=m
 CONFIG_REGULATOR_QCOM_SPMI=m
-CONFIG_BT_HCIUART=m
-CONFIG_BT_HCIUART_QCA=y
 CONFIG_DRM_MSM=m
 # CONFIG_DRM_MSM_DSI_28NM_PHY is not set
 # CONFIG_DRM_MSM_DSI_20NM_PHY is not set
